src/main/java/org/codewars/utils/Reader.java

```diff
@@ -1,8 +1,10 @@
 package org.codewars.utils;
 
 import java.math.BigInteger;
+import java.util.Scanner;
 
 public class Reader {
+    static Scanner scanner = new Scanner(System.in);
```

Comment:
we with @DappY-127 started use this:
private static final Scanner scanner = new Scanner(System.in);